### PR TITLE
Codepen button should show only when resources loaded

### PR DESCRIPTION
**Pull request by spectranaut**
It was a good point @smhigley to show the button after the resources have been loaded instead of hiding it if they don't! :)

I didn't implement the promise array because [XMLHttpRequest.send](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/send) doesn't return a promise but that it's true that would have been much cleaner.

### CODE REVIEW

File: examples/js/examples.js [hunk 1]
```diff
@@ -378,32 +380,28 @@ function addOpenInCodePenForm (exampleIndex, exampleHeaderId, exampleCodeId, exa
               totalFetchedFiles++;
           }
           else {
-              hideButton(buttonId, "Could not load resource: " + href);
+              console.log("Not showing 'Open in Codepen' button. Could not load resource: " + href);
```
Comment (nschonni) [inline examples/js/examples.js:383]: Does it make sense to warn/error for these?
```suggestion
              console.warn("Not showing 'Open in Codepen' button. Could not load resource: " + href);
```

Comment (spectranaut) [reply examples/js/examples.js]: makes sense! pushed commit for switch.

**Comment by mcking65** [comment]
@smhigley friendly ping when you get time

**Review by smhigley — APPROVED** [review]
love it! :)

**Review by carmacleod — APPROVED** [review]
+1

Looks good!

**Comment by mcking65** [comment]
Thank you @smhigley  and @carmacleod  ... will merge!

**Pull request merged**